src/praisonai-agents/praisonaiagents/agent/execution_mixin.py

```
@@ -918,6 +918,75 @@ def run_chat():
+        for tool in self.tools:
+            name = getattr(tool, '__name__', None)
+            if name:
+                existing_names.add(name)
+
+        try:
+            from praisonaiagents.tools.skill_tools import (
+                skill_manage,
+                read_skill_file,
+                list_skill_scripts,
+            )
+        except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)